Dockerfile

```
@@ -20,12 +20,12 @@ WORKDIR $GOPATH/src/github.com/apptainer
 RUN git clone https://github.com/apptainer/apptainer.git \
     && cd apptainer \
     && git checkout "$APPTAINER_COMMITISH" \
-    && ./mconfig -p /usr/local/apptainer \
+    && ./mconfig --without-suid -p /usr/local/apptainer \
```

@peter-jansson - i would be more comfortable building with suid for now. most people running docker will be running it as root (because rootless docker is a relatively new feature), so the suid shouldn't be a problem. in the future, i would be more comfortable building without suid. and like you said in a separate comment, this could be an argument. for now, though, could you change this to `--with-suid` ? i have added a code suggestion to this comment.

after that, this PR look good to merge on my end.

```suggestion
    && ./mconfig --with-suid -p /usr/local/apptainer \
```

---

Absolutely, will do.